src/frontend/src/env/dapp-descriptions.json

```diff
@@ -138,7 +138,7 @@
 	},
 	{
 		"id": "aave",
-		"name": "aave",
+		"name": "Aave",
```

Comment:
even if it is a small change, it is always better to separate in another PR, since it is another topic

Reply:
No we cannot capitalize. See zkPoker which is displayed with small letter first.

Reply:
Gotcha! Thanks for the explanation